contracts/integrations/test/actors/pool_operator.ts

```diff
@@ -0,0 +1,60 @@
+import { TransactionReceiptWithDecodedLogs } from 'ethereum-types';
+
+import { Actor, ActorConfig, Constructor } from './base';
+
+export interface PoolOperatorConfig extends ActorConfig {
+    operatorShare: number;
+}
+
+export function PoolOperatorMixin<TBase extends Constructor>(Base: TBase) {
+    return class extends Base {
+        public readonly operatorShare: number;
+        public readonly poolIds: string[] = [];
```

Comment:
It would make more sense to me for this to just be `poolId` since an operator is only the operator of a single pool. Can you explain this design choice?

Reply:
I don't think that's actually the case; a single address can serve as the operator of multiple pools, it just can't be a maker in more than one

Reply:
Yep, you're completely right.